x-pack/plugins/stack_alerts/public/alert_types/es_query/expression/es_query_expression.tsx

```diff
@@ -208,7 +209,6 @@ export const EsQueryExpression: React.FC<
           languageId="xjson"
           width="100%"
           height="200px"
-          data-test-subj="queryJsonEditor"
```

Comment:
CodeEditor UI Component doesn't pass the data-test-subj attribute to the final rendered html, so i moved it to the parent row element